Interop/Indexer/src/main/kotlin/org/jetbrains/kotlin/native/interop/indexer/J2ObjCParser.kt

```diff
@@ -104,6 +103,26 @@ class J2ObjCParser: ClassVisitor(Opcodes.ASM5) {
     }
   }
 
+  /**
+   * Parses the methods in the class to find getters/setters and generates properties accordingly
+   *
+   * @param methods List of ObjCMethods in the class
+   * @return List of ObjCProperty based off of the getters/setters in class
+   */
+  private fun parseForProperties(methods: List<ObjCMethod>): List<ObjCProperty> {
+    val getters = methods.filter{(it.selector.startsWith("get")) && it.parameters.isEmpty()}
```

Comment:
Keep in mind this is very partial: getters sometimes start with isXxx(), or even just xxx().  It's ok to just handle get for now but I'd probably make a TODO to generalize.

I'd probably also check that the return type isn't void, FWIW. It may also matter if this is a static method? Static getters can still be properties, conceivably, but they'd presumably be called differently.

Reply:
Oh also you might consider ignoring private properties.  For non-private, I'm not sure if the exact visibility matters?

Reply:
+1 for ignoring privates

I think adding a good todo here is critical.

I think Kotlin only extras properties for getXXX and setXXX and isXXX for boolean.  (as per https://kotlinlang.org/docs/reference/java-interop.html#getters-and-setters)

Reply:
Added a TODO and added "is" to the checks. 

Private/Public checks were part of the next set of features planned so I can't add that check as of now.